### ISSUE TITLE
use confu env settings util

### ISSUE THREAD

**Issue by vegu** [fullctl/aaactl#14]

**Comment by egfrank** [comment]
I had to add `confu 1.7.1` to the poetry file here and now when trying to rebuild my Docker container, it's hanging on 

```
Step 22/46 : RUN poetry install --no-root
 ---> Running in 3d022c1d7e08
Installing dependencies from lock file

Package operations: 89 installs, 0 updates, 0 removals

  • Installing certifi (2020.12.5)
  • Installing chardet (4.0.0)
  • Installing idna (2.10)
  • Installing pycparser (2.20)
  • Installing urllib3 (1.26.4)
  • Installing cffi (1.14.5)
  • Installing defusedxml (0.7.1)
  • Installing oauthlib (3.1.0)
  • Installing requests (2.25.1)
  • Installing vine (5.0.0)
  • Installing wcwidth (0.2.5)
  • Installing amqp (5.0.6)
  • Installing click (7.1.2)
  • Installing cryptography (3.4.7)
  • Installing future (0.18.2)
  • Installing prompt-toolkit (3.0.18)
  • Installing pyjwt (2.0.1)
  • Installing pyparsing (2.4.7)
  • Installing python3-openid (3.2.0)
  • Installing pytz (2021.1)
  • Installing pyyaml (5.4.1)
  • Installing requests-oauthlib (1.3.0)
  • Installing six (1.15.0)
  • Installing smmap (4.0.0)
  • Installing sqlparse (0.4.1)
```

**Comment by egfrank** [comment]
PR is here but it isn't tested because of the Docker issue above
https://github.com/fullctl/aaactl/pull/15

**Comment by egfrank** [comment]
@vegu Just ran tests on the PR and linted it so it should be ready for code review!

**Comment by grizz** [comment]
Fixed with #15
